build/eslint/xfail.list

```diff
@@ -1,298 +1,206 @@
+apps/email/build/main-frame-setup.build.js:3:6: Parsing error: Unexpected token [Error]
+apps/email/js/ext/ext/bleach.js/index.js:8:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/email/js/html_cache_restore.js:397:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/email/js/template.js:50:7: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/email/js/template.js:177:9: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/email/js/tmpl.js:8:9: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/email/test/unit/message_reader_card_test.js:121:7: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/ftu/js/wifi.js:351:7: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/ftu/test/unit/language_test.js:21:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/ftu/test/unit/language_test.js:34:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/gallery/test/marionette/apps/galleryactivitytester/js/galleryactivitytester.js:32:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/gallery/test/marionette/apps/galleryactivitytester/js/galleryactivitytester.js:33:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/gallery/test/marionette/apps/galleryactivitytester/js/galleryactivitytester.js:34:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/keyboard/build/settings-config.js:61:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/keyboard/build/settings-config.js:72:5: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/keyboard/js/imes/jspinyin/tests/init.js:103:3: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/keyboard/js/imes/jszhuyin/lib/jszhuyin_server.js:42:7: Using postMessage() with wildcard targets is not allowed. [Error/no-wildcard-postmessage/no-wildcard-postmessage]
+apps/keyboard/js/views/emoji_key_view.js:22:3: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/keyboard/js/views/key_view.js:118:3: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/keyboard/js/views/key_view.js:127:3: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/keyboard/js/views/key_view.js:134:3: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/keyboard/js/views/key_view.js:139:3: Unsafe assignment to innerHTML [Error/no-unsafe-innerhtml/no-unsafe-innerhtml]
+apps/keyboard/test/unit/keyboard/layout_normalizer_test.js:64:30: Parsing error: Unexpected token [Error]
```

Comment:
this is the array comprehension syntax; I don't know if a newer babel could parse it ?